scripts/ci/upload-release

```diff
@@ -50,7 +50,7 @@ DCOS_BRANCH="dcos-ui/${PKG_TARGET}+dcos-ui-latest"
 VERSION_PLACEHOLDER="0.0.0-dev+semantic-release"
 
 # first, replace placeholder in dist bundle and verify that it worked
-sed -i "" "s/${VERSION_PLACEHOLDER}/${PKG_TARGET}+${PKG_VERSION}/" ${PROJECT_ROOT}/dist/index.html
+sed -i "s/${VERSION_PLACEHOLDER}/${PKG_TARGET}+${PKG_VERSION}/" ${PROJECT_ROOT}/dist/index.html
```

Comment:
Will it work in osx? Shall we abandon in-place change and go with
```bash
sed -i "s/${VERSION_PLACEHOLDER}/${PKG_TARGET}+${PKG_VERSION}/" ${PROJECT_ROOT}/dist/index.html ${PROJECT_ROOT}/dist/index.html
```

Reply:
`sed -i''` actually works on macOS 🤔 changed it to that is works on linux&macOS